agent/exploits/cve_2024_21762.py

```diff
@@ -82,7 +82,7 @@ def check(self, target: definitions.Target) -> list[definitions.Vulnerability]:
         try:
             socket_wrapper.read(2048)
         except socket.error as exc:
-            if type(exc) == TimeoutError:
+            if type(exc) is TimeoutError:
```

Comment:
Why aren't we doing except TimeoutError

Reply:
i am not sure, this is from an old CVE i modify it just to solve a mypy workflow error